src/System.Net.NetworkInformation/tests/FunctionalTests/NetworkInterfaceBasicTest.cs

```diff
@@ -37,7 +37,16 @@ public void BasicTest_AccessInstanceProperties_NoExceptions()
                 _log.WriteLine("Type: " + nic.NetworkInterfaceType);
                 _log.WriteLine("Status: " + nic.OperationalStatus);
                 _log.WriteLine("Speed: " + nic.Speed);
-                Assert.True(nic.Speed >= 0, "Overflow");
+
+                if (nic.OperationalStatus != OperationalStatus.Down)
+                {
+                    Assert.True(nic.Speed >= 0, "Overflow");
```

Comment:
As long as you're changing this, I'd suggest making it:
```C#
Assert.InRange(nic.Speed, 0, long.MaxValue);
```
That way if it does fail, the assert message will include the value that was out of range.